dbms/include/DB/DataStreams/MergingSortedBlockInputStream.h

```diff
@@ -57,9 +58,10 @@ class MergingSortedBlockInputStream : public IProfilingBlockInputStream
 {
 public:
 	/// limit - если не 0, то можно выдать только первые limit строк в сортированном порядке.
-	MergingSortedBlockInputStream(BlockInputStreams inputs_, const SortDescription & description_, size_t max_block_size_, size_t limit_ = 0)
+	MergingSortedBlockInputStream(BlockInputStreams & inputs_, const SortDescription & description_,
+								  size_t max_block_size_, size_t limit_ = 0, MergedRowSources * row_sources_ = nullptr)
```

Comment:
No comment about what `row_sources` is.
No comment about that it is out parameter. You could name it out_row_sources for better clarity.